src/server/v2.0/handler/preheat.go

```diff
@@ -80,3 +86,128 @@ func convertProvidersToFrontend(backend []*provider.Metadata) (frontend []*model
 	}
 	return
 }
+
+// GetPolicy is Get a preheat policy
+func (api *preheatAPI) GetPolicy(ctx context.Context, params operation.GetPolicyParams) middleware.Responder {
```

Comment:
Don't forget to add RBAC controlling later.

Reply:
I'll add RBAC after this merged.